### DISCUSSION TITLE
Translation hotkey vocabulary problem

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#432] [General]
![螢幕快照 2022-12-04 上午12 26 04](https://user-images.githubusercontent.com/44593430/206842548-c20e40df-d789-4ebc-9074-f2f1af33ee26.png)
![螢幕快照 2022-12-04 上午12 25 47](https://user-images.githubusercontent.com/44593430/206842543-51c80db2-53d5-410b-a86c-c7c8e07a0110.png)
Logically speaking, it should be like building blocks on Hotkey
"%1"+"%2"+"%3"+"%4"
But here it is a group of translations, there will be a large number of single groups in the translation, and then 27 languages also have such strings, think about how big the file will be.
This is only the translation string of filer.

**Comment by probonopd** [comment]
What should we do differently?

**Comment by louies0623** [comment]
> What should we do differently?

Modularize them,as shown
![think_工作區域 1](https://user-images.githubusercontent.com/44593430/207525190-0490b03e-b425-48f9-bec2-3f26fd58cbe4.png)

**Comment by probonopd** [comment]
I still don't quite understand what the issue is, but in general I think shortcuts should not be translatable, as they should be the same no matter what language the user has selected.

For example, the German word for "Copy" stars with "K" but the shortcut is still "C".

**Reply by louies0623** [reply]
I agree, there is just one thing I want to say, because the key names are replaced by symbols, like when I use MacOS or hello os, I have to look up what the symbols are all the time, wasting a lot of time looking up symbols, I hope there is a PC keyboard  Display method
